libs/design/guides/getting-started.md

```diff
@@ -1,96 +1,74 @@
-  @fortawesome/free-solid-svg-icons@^5.2.0 \
-  @fortawesome/free-brands-svg-icons@^5.2.0 \
-  @fortawesome/free-regular-svg-icons@^5.2.0 \
-  modern-normalize@^0.5.0
+npm install @daffodil/design --save 
 ```
 
-> If you are using Node.js v14 or higher, you can simply run: `npm install --save @daffodil/design`
-
-### Yarn
+Install with Yarn:
 ```bash
-yarn add @daffodil/design @daffodil/core \
-  @angular/animatio@^13.0.0 \
-  @angular/cdk@^13.0.0 \
-  @fortawesome/angular-fontawesome@^0.10.0 \
-  @fortawesome/fontawesome-svg-core@^1.0.0 \
-  @fortawesome/free-solid-svg-icons@^5.2.0 \
-  @fortawesome/free-brands-svg-icons@^5.2.0 \
-  @fortawesome/free-regular-svg-icons@^5.2.0 \
-  modern-normalize@^0.5.0
+yarn add @daffodil/design
 ```
 
-> If you are using Node.js v14 or higher, you can simply run: `yarn add @daffodil/design`
-
-### Modify Angular.json
-Next, create a `styles.scss` in the root of the Angular app and modify the `angular.json`'s `projects.[my-app].architect.build.options.styles` to look as below. If SASS is already in use in the application and the stylesheet already exists, this step can be skipped.
+## Set up your style environment
+1. Create a `styles.scss` in the root of your Angular app.
+2. Add the stylesheet to your `angular.json` file. This step can be skipped if SASS is already in use in the application and the stylesheet already exists.
 
 ```json
-"styles": [
-  "src/styles.scss"
-],
+"projects": {
+  "app-name": {
+    "architect": {
+      "build": {
+        "options": {
+          "styles": [
+            "src/styles.scss"
+          ],
+        }
+      }
+    }
+  }
+}
 ```
 
-> Note that `@daffodil/design` uses SASS, [so we recommend you learn it!](https://sass-lang.com/)
-
-### Add the Global Styles
-There is a minimal required global style for `@daffodil/design` to operate effectively in all supported browsers. Update the `styles.scss` to include the following:
+### Add global styles
+There is a minimal required global style for the Daffodil Design System to operate effectively in all supported browsers. Update your `styles.scss` file to include the following:
 
 ```scss
 @forward '@daffodil/design/scss/global';
 ```
 
-> For more information on our approach to these kinds of styles, see the ["Global Styles" guide.](/libs/design/guides/global-styles.md)
+> For more information on our approach to these kinds of styles, see the [Global Styles guide.](/libs/design/guides/foundations/global-styles.md)
 
-### Add a Theme
-`@daffodil/design` is a themable component library. The components in the design library can be configured with customized colors in addition to a dark AND light mode for those same colors.
+## Add a theme
+`@daffodil/design` is a themable component library. The components in the design library can be configured with customized colors in addition to a dark and light mode for those same colors. No further configuration is needed if you choose to use `@daffodil/design`'s default theme.
```

Comment:
Is this correct? Don't you still need to add the theme `@use`?

Reply:
updated